Update dashboard-teacher.html

camiP89 opened the pull request:
### What does this PR do? 

### Description of Task to be completed? I have re-written in the HTML, and styled the dashboard to look like the Figma design. I have added the links. The task was to work on the layout and navigation. Added the function for the burger menu. 

### How should this be manually tested? You can test the links and the burger menu. 

### Any background context you want to provide?

camiP89 committed:
Update dashboard-teacher.html

kevindjenhansen commented:
Could you provide more context? :)

What does this PR do?
Description of Task to be completed?
How should this be manually tested?
Any background context you want to provide?

camiP89 commented:
I have re-written in the HTML, and styled the dashboard to look like the Figma design. I have added the links. The task was to work on the layout and navigation. Added the function for the burger menu. You can test the links and the burger menu.

kevindjenhansen reviewed (changes requested):
Amazing work so far, here are a few things I noticed that could be improved on:

- On desktop view make sure you are using the correct icon, I noticed you are using "/public/img/Student initials mobile.png" when you should be using "/public/img/Student initials.png"
- You should add more spacing from the header and the main content 
- I've noticed that when you have the sidebar it pushes the main content to the side - make sure everything stays centered regardless of if the sidebar is there
- Add the correct font and bold weight onto the "Teacher at Edugate School" and button texts - if you go on the Figma design file you can click on the text then on the side it will show you which font it uses and the weight
- On mobile increase the size of the "Dashboard" text
- On mobile increase the gap between the JB-icon and the "Joe Bloggs" text to gap-4
- On mobile the footer is hovering above the screen, make sure the footer is always at the bottom

camiP89 committed:
corrected mobile icon to desktop icon. Added more spacing between header and main content. Changed font and font weight on "Teacher at Edugate School" and button texts. Increased the size of the Dashboard text. Increased gap between JB icon and the "Joe Bloggs" text to gap-4. Taken away top margin on footer.

corrected mobile icon to desktop icon. Added more spacing between header and main content. Changed font and font weight on "Teacher at Edugate School" and button texts. Increased the size of the Dashboard text. Increased gap between JB icon and the "Joe Bloggs" text to gap-4. Taken away top margin on footer.

kevindjenhansen reviewed (changes requested):
- The images for the buttons (Top students, Add results, View results, See profile, Log out) don't seem to be centered with the text
- Did you change the header to the one in the header.html file? It looks a bit different. But if you could that way we have a consistent and uniform header between all the pages

Flisa81 commented:
> * The images for the buttons (Top students, Add results, View results, See profile, Log out) don't seem to be centered with the text
> * Did you change the header to the one in the header.html file? It looks a bit different. But if you could that way we have a consistent and uniform header between all the pages

Hi!
I have centered the layout. The header was not assigned to me

camiP89 committed:
Update dashboard-teacher.html

camiP89 committed:
Merge branch 'develop' into teacher-dashboard-edugate

camiP89 committed:
Update dashboard-teacher.html

camiP89 committed:
Merge branch 'develop' into teacher-dashboard-edugate

kevindjenhansen reviewed (approved):
Great work!
